modules/web-discovery-project/sources/web-discovery-project.es

```diff
@@ -1642,6 +1642,13 @@ const WebDiscoveryProject = {
     config_ts: null,
     config_location: null,
   },
+  whitelist: [
+    /play\.google\.com\/store\/apps\/details\?id=[^\/]+$/,
+    /apps\.apple\.com\/\D{2}\/app\/[\w-]+\/id\d+$/,
+    /www\.npr\.org\/\d{4}\/\d{2}\/\d{2}/,
+    /variety\.com\/\d{4}\//,
```

Comment:
Would it be safer to include a prefix match including protocol here? e.g. `https://play.google.com/store/apps/details?id=...`